From 412191da499f4f96dd9138a2868b16947b8d8f70 Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Mon, 19 Jul 2021 10:31:05 -0700
Subject: [PATCH] Avoid void pointer arithmetic

This is not strictly allowed in ISO C, which requires the pointed-to
type to be a complete object type.
---
 bfd/peicode.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bfd/peicode.h b/bfd/peicode.h
index c84b135e1eb..41955975cdc 100644
--- a/bfd/peicode.h
+++ b/bfd/peicode.h
@@ -1474,7 +1474,7 @@ pe_bfd_object_p (bfd * abfd)
   if (opt_hdr_size != 0)
     {
       bfd_size_type amt = opt_hdr_size;
-      void * opthdr;
+      bfd_byte * opthdr;
 
       /* PR 17521 file: 230-131433-0.004.  */
       if (amt < sizeof (PEAOUTHDR))
-- 
2.32.0

